OsmIntegrator.Tests/Helpers/TileHelper.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Newtonsoft.Json;
+using OsmIntegrator.ApiModels;
+using OsmIntegrator.ApiModels.Auth;
+using OsmIntegrator.Tests.Helpers.Base;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace OsmIntegrator.Tests.Helpers
+{
+    public class TileHelper : BaseHelper
+    {
+        private List<string> _tileIdList;
+            
+        private LoginData _defaultLoginData = new LoginData
+        {
+            Email = "supervisor1@abcd.pl",
+            Password = "12345678",
+        };
+
+        public Dictionary<int, Tile> GetTestTileDict()
+        {
+            //var list = new List<string>
```

Comment:
Cóż to za komentarz Marcin, czy on jest konieczny?

Reply:
Nie. Następnym razem go usunę.